lib/ReactViews/Map/Navigation/MapNavigation.tsx

```diff
@@ -289,9 +289,18 @@ class MapNavigation extends React.Component<PropTypes> {
                 `margin-bottom: 5px;`}
             `}
           >
-            {items.map(item => (
-              <MapNavigationItem key={item.id} item={item} terria={terria} />
-            ))}
+            {items.map(item => {
+              // Do not expand in place for horizontal orientation
+              // as it results in buttons overlapping and hiding neighboring buttons.
+              return (
+                <MapNavigationItem
+                  expandInPlace={this.orientation !== Orientation.HORIZONTAL}
```

Comment:
this seems a bit odd to me but eh, its better than overlapping i guess

Reply:
This can cause an issue with buttons going of screen
![Screenshot_20210923_084059.jpg](https://user-images.githubusercontent.com/19208948/134464521-e4932565-034c-4e77-9375-d54b917785fd.jpg)

Reply:
That's a good point @zoran995 - I have filed an issue here https://github.com/TerriaJS/terriajs/issues/5847 and will bring it to @philipgrimmett's attention.

I think this is a better alternative than the current behavior so I'll go with this change for now until we have a more permanent fix.

Reply:
we can look into updating MapNavigation `computeSizes` and `updateNavigation`, to run in more cases (when buttons are expanded or similar)

Reply:
I've added a small fix to add flex-wrap in the container which will at least make the icons appear on screen. However there is a bad interaction with hover if the expanded button is too big and wraps around. I've added the detail into #5847